portfolio/src/test/java/com/google/sps/TaskQueueTest.java

```diff
@@ -66,7 +65,153 @@ public void checkCorrectSortingShortestTaskFirst() {
 
     TaskQueue taskQueue = new TaskQueue(taskList, SHORTEST_TASK_FIRST);
 
-    for (int i = 0; i < expected.size(); i++) {
+    int i = 0;
+    while (i++ < expected.size()) {
```

Comment:
We can introduce a method to taskQueue like isEmpty(), and then here do,

while (!taskQueue.isEmpty()) {

Reply:
Done.